hy/core/bootstrap.hy

```diff
@@ -7,7 +7,7 @@
 ;;; They are automatically required everywhere, even inside hy.core modules.
 
 (defmacro if [&rest args]
-  "if with elif"
+  "Conditionally evaluate `args`, when given three conditions equiv. to `elif`."
```

Comment:
This is a comma splice, if I'm reading it correctly.

Reply:
I misread the documentation, I've never used `if` with more than one condition. I'll fix that up.